### PR TITLE
Format all PO files with `dprint`

### PR DESCRIPTION
**Pull request by mgeisler**
This enforces a consistent formatting for the PO files. The goal of this is to avoid large diffs due to random and unnecessary reformatting.

We use the format of `msgcat`: this is also waht `msgmerge` produces and it’s easy to replicate for people by installing Gettext and running `dprint fmt`.

This is a follow-up to #1351 which started enforcing that the `msgid` fields don’t change due to reformatting in a PR.

If this turns out to be cumbersome, then we can disable it again.

### CODE REVIEW

File: dprint.json [hunk 1]
```diff
@@ -8,6 +8,9 @@
     "commands": [{
       "command": "rustfmt --edition 2021",
       "exts": ["rs"]
+    }, {
+      "command": "msgcat -",
```
Comment (sakex) [inline dprint.json:12]: How does this work? Is it piping rustfmt output to msgcat ?

**Review by sakex — APPROVED** [review]

File: dprint.json (hunk 1 above)
Comment (mgeisler) [reply dprint.json:12]: Almost, it's piping all `.po` files (from `exts` below) through `msgcat -`. This normalizes the PO files.

**Comment by mgeisler** [comment]
Note that this will require translators to install the Gettext tools and `dprint` so they can run `dprint fmt` before sending out a file for review. I've tried to avoid this until now since I wanted to make it as easy as possible to contribute to a translation. But when the Poeditors don't format things consistently, we end up with a mess.

**Comment by henrif75** [comment]
> Note that this will require translators to install the Gettext tools and `dprint` so they can run `dprint fmt` before sending out a file for review. I've tried to avoid this until now since I wanted to make it as easy as possible to contribute to a translation. But when the Poeditors don't format things consistently, we end up with a mess.

We'll probably need to update the instructions in TRANSLATE.md, right?

**Comment by mgeisler** [comment]
> We'll probably need to update the instructions in TRANSLATE.md, right?

Yes! Thanks, that's a great point!

**Comment by mgeisler** [comment]
@henrif75 what do you think of the update to TRANSLATIONS?

**Comment by henrif75** [comment]
> @henrif75 what do you think of the update to TRANSLATIONS?

Sure, I'll do it.

**Comment by henrif75** [comment]
> @henrif75 what do you think of the update to TRANSLATIONS?

They're great, thanks!

**Pull request merged**